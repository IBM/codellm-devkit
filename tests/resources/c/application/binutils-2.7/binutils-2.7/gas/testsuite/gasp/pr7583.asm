
     .sdata "v1.0000"
     .end
