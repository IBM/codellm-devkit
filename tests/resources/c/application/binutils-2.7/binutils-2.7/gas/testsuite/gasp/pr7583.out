!

!     .sdata "v1.0000"
	.byte	118,49,46,48,48,48,48
!     .end
